content/en/developers/ide_integrations/idea/_index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Datadog Plugin For IntelliJ IDEA
+kind: documentation
+disable_toc: false
+is_beta: true
+further_reading:
+- link: "/account_management/api-app-keys/"
+  tag: "Documentation"
+  text: "Find out more about API and application keys."
+- link: "/getting_started/profiler/"
+  tag: "Documentation"
+  text: "Getting started with Continuous Profiler."
+---
+
+{{< beta-callout url="#" btn_hidden="true">}}
+  The Datadog plugin for IntelliJ IDEA is in Public Beta. It is intended for customers that use the <a href="https://docs.datadoghq.com/profiler/#pagetitle">Continuous Profiler</a> for their Java services. If the plugin stops working unexpectedly, check for plugin updates.
+{{< /beta-callout >}}
+
+## Overview
+
+The Datadog plugin for IntelliJ IDEA helps improve software performance by providing meaningful code-level insights in the IDE based on real-time observability data. The plugin helps reduce latency and lower cloud costs by highlighting code lines that:
+- consume the most CPU
+- allocate the most memory
+- spend the most time on locks, disk I/O, socket I/O, and so on
+
+{{< img src="/developers/ide_integrations/idea/overview.png" alt="The Datadog tool window open in IDEA" style="width:100%;" >}}
+
+The plugin features:
+- Top List: Identify methods that consume the most resources
```

Comment:
Add something along these lines:
Navigate directly from Top List, Flame graph to the respective code line